src/Parser.js

```diff
@@ -44,22 +45,23 @@ export function parse (value, points, declarations, rules, rulesets) {
 				break
 			// \t \n \s
 			case 9: case 10: case 32:
-				temporary += whitespace(previous)
+				temporary += variable ? whitespace(previous) : from(character)
```

Comment:
we don't need this.

Reply:
technically we do, if we want to be 100% correct - when reading css variables values in JS this whitespace matters, while preserving trailing & leading whitespace doesnt really seem important for real-life, im unsure if this would handle OK all the whitespace **in** the value if we remove this, and the whitespace in the value seems important, i can test this tomorrow to see how the whitespace would be handled then, but maybe you know from the top of your head?

Reply:
In what case would the whitespace be important?

Reply:
https://jsfiddle.net/haqxs9ug/

Reply:
>"       white    space   " -> `" white space "`

The browser strips most of the whitespace, how is whitespace important in that fiddle?

Reply:
most - yes, but not all, this changed line makes leading & trailing whitespaces preserved (including others as well, but those will be stripped by browsers anyway - like you have mentioned)

Reply:
Yes, but like CSS i don't see how this is important in CSS. Do you have any case where this has an observable behavior in CSS?

For example consider this:

```css
--rgb: 0, 0, 0;
--rgba rgba(var(rgb), 0.1);
```

Whether it's `0, 0, 0` or `0,0,0` the effect will be the same.

Reply:
nah, cant see any problems with CSS, just when reading with JS, which is also far-fetched